components/Card.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import Image from "next/image";
+
+const Card = ({ item }) => {
+  return (
+    <div className="flex justify-center mt-5">
+      <div className="rounded-lg shadow-xl bg-white max-w-[300px]  lg:max-w-xs xl:max-w-sm ">
+        <a href="#!">
+          <Image
```

Comment:
Use responsive design approach, see https://tailwindcss.com/docs/responsive-design

Reply:
Test on small and bigger browser screen sizes

Reply:
can you deploy and check on bigger screens. Read docs on how to make it responsive. I can't see first and last card properly on my screen. 

![image](https://user-images.githubusercontent.com/208305/217500671-4061d1fc-8928-4c48-909b-152292906d28.png)